src/Display/WebGl/simple_server.py

```diff
@@ -51,6 +51,24 @@ def get_available_port(port):
     return port
 
 
+def get_interface_ip(family: socket.AddressFamily) -> str:
+    """Get the IP address of an external interface. Used when binding to
+    0.0.0.0 or ::1 to show a more useful URL. Inspire of `werkzeug`.
+
+    :meta private:
+    """
+    # arbitrary private address
+    host = "fd31:f903:5ab5:1::1" if family == socket.AF_INET6 else "10.253.155.219"
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Consider using well-known IP addresses for testing connectivity.

Using arbitrary private IP addresses might not be reachable in all environments. Consider using well-known IP addresses like '8.8.8.8' for IPv4 or '2001:4860:4860::8888' for IPv6.

```suggestion
host = "2001:4860:4860::8888" if family == socket.AF_INET6 else "8.8.8.8"
```